tests/services/inventory/test_data/additional_cai_org_policies.dump

```diff
@@ -0,0 +1,3 @@
+{"name":"//cloudresourcemanager.googleapis.com/organizations/111222333","asset_type":"cloudresourcemanager.googleapis.com/Organization","org_policy":[{"constraint":"constraints/compute.disableSerialPortAccess","update_time":{"seconds":1539891911,"nanos":215000000},"boolean_policy":{"enforced":true}}],"ancestors":["organizations/111222333"]}
```

Comment:
Same here, is this file being used?

Reply:
Removed file. Mock files are being used.